genny/newapp/core/core.go

```diff
@@ -30,14 +29,14 @@ func New(opts *Options) (*genny.Group, error) {
 	app := opts.App
 
 	if app.WithModules {
-		g, err := gomods.Init(app.PackagePkg, app.Root)
-		if err != nil {
-			return gg, err
-		}
-		g.Command(gogen.Get("github.com/gobuffalo/buffalo@" + runtime.Version))
-		g.Command(gogen.Get("./..."))
-
-		gg.Add(g)
+		// g, err := gomods.Init(app.PackagePkg, app.Root)
```

Comment:
Is this still necessary after the latest fix on master?

Reply:
I don't think is necesary, I'm closing this PR in favor of #1676.